sdk/src/types/block/semantic/mod.rs

```diff
@@ -308,6 +385,12 @@ impl<'a> SemanticValidationContext<'a> {
             }
         }
 
+        for (_, (account_input_mana, account_output_mana)) in &self.block_issuer_mana {
+            if self.input_mana - account_input_mana < self.output_mana - account_output_mana {
```

Comment:
we are absolutely sure this can never panic and is strictly less?

Reply:
Yes, rhs is a subset of lhs